### PR TITLE
Target DotSpatial.Projections to netstandard 2.0

### PR DESCRIPTION
**Pull request by inforithmics**
Fixes # .

### Checklist

- [x] I have included examples or tests
- [ ] I have updated the change log
- [x] I am listed in the CONTRIBUTORS file
- [x] I have cleaned up the commit history (use rebase and squash)

### Changes proposed in this pull request:

- Add Target for netstandard for DotSpatial Projection

### CODE REVIEW

**Comment by inforithmics** [comment]
I fixed the nuget package creation now the unit tests ned to be fixed

**Comment by inforithmics** [comment]
All Tests should now be green, what remains is my initial goal to add a .netstandard 2.0 target

**Comment by inforithmics** [comment]
Heureka so the MoveDsToNet6 is green with this checkin

**Comment by inforithmics** [comment]
I could only sqash it to 5 commites

File: Source/.nuget/Pack.bat [hunk 1]
```diff
@@ -19,7 +19,7 @@ nuget pack ..\DotSpatial.NTSExtension\DotSpatial.NTSExtension.csproj -version "%
 nuget pack ..\DotSpatial.Positioning\DotSpatial.Positioning.csproj -version "%NugetVersion%" -Properties "PackageVersion=%PackageVersion%";"Configuration=Release" 
 nuget pack ..\DotSpatial.Positioning.Design\DotSpatial.Positioning.Design.csproj -version "%NugetVersion%" -Properties "PackageVersion=%PackageVersion%";"Configuration=Release" 
 nuget pack ..\DotSpatial.Positioning.Forms\DotSpatial.Positioning.Forms.csproj -version "%NugetVersion%" -Properties "PackageVersion=%PackageVersion%";"Configuration=Release" 
-nuget pack ..\DotSpatial.Projections\DotSpatial.Projections.csproj -version "%NugetVersion%" -Properties "PackageVersion=%PackageVersion%";"Configuration=Release"
```
Comment (jany-tenaj) [inline Source/.nuget/Pack.bat:22]: Why did you switch from csproj to nuspec?
I'd rather get rid of nuspec completely.

File: Source/Core/DotSpatial.Data.Forms/DotSpatial.Data.Forms.csproj [hunk 2]
```diff
@@ -12,4 +13,31 @@
     <ProjectReference Include="..\DotSpatial.Symbology\DotSpatial.Symbology.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
```
Comment (jany-tenaj) [inline Source/Core/DotSpatial.Data.Forms/DotSpatial.Data.Forms.csproj:16]: Why did you add those lines?
VS 2022 shows this information without having to add it to the csproj file.

File: Source/Core/DotSpatial.Data.Forms/DotSpatial.Data.Forms.csproj [hunk 3]
```diff
@@ -12,4 +13,31 @@
...
+
+  <ItemGroup>
+    <EmbeddedResource Update="DataFormsMessageStrings.resx">
+      <Generator>ResXFileCodeGenerator</Generator>
+      <LastGenOutput>DataFormsMessageStrings.Designer.cs</LastGenOutput>
+    </EmbeddedResource>
+    <EmbeddedResource Update="DialogImages.resx">
+      <Generator>ResXFileCodeGenerator</Generator>
+      <LastGenOutput>DialogImages.Designer.cs</LastGenOutput>
+    </EmbeddedResource>
```
Comment (jany-tenaj) [inline Source/Core/DotSpatial.Data.Forms/DotSpatial.Data.Forms.csproj:40]: Why did you add those lines?
VS 2022 shows this information without having to add it to the csproj file.

File: Source/Core/DotSpatial.NTSExtension/TopologyText.Designer.cs [hunk 4]
```diff
@@ -1,50 +1,45 @@
 ﻿//------------------------------------------------------------------------------
 // <auto-generated>
 //     This code was generated by a tool.
-//     Runtime Version:4.0.30319.34014
+//     Runtime Version:4.0.30319.42000
 //
 //     Changes to this file may cause incorrect behavior and will be lost if
 //     the code is regenerated.
 // </auto-generated>
 //------------------------------------------------------------------------------
```
[the remark this answers is not in the document]
Comment (inforithmics) [reply Source/Core/DotSpatial.NTSExtension/TopologyText.Designer.cs:10]: The reason this was changed is that I connected the .resx file with the ResourceGenenerator and then the file got regenerated with Visual Studio 2022 so the version number changed. So I can revert this.

[the remark this answers is not in the document]
Comment (inforithmics) [reply Source/Core/DotSpatial.NTSExtension/TopologyText.Designer.cs:10]: I mean the Runtime Version ... change but not the connection.

File: Source/Core/DotSpatial.Data.Forms/DotSpatial.Data.Forms.csproj [hunk 5]
```diff
@@ -12,4 +13,31 @@
...
+  </ItemGroup>
+
```
[the remark this answers is not in the document]
Comment (inforithmics) [reply Source/Core/DotSpatial.Data.Forms/DotSpatial.Data.Forms.csproj:42]: This lines where added because the .Designer Files are only generated when the .resx file is connected to the ResxFileCodeGenerator, this is something that is not done automatically. And because the ResxFileGenenerator was not connected with with the resx File the old Generated Designer.cs Code didn't match the location and namespaces and didn't find the resources which caused exceptions when Resources where accessed.

File: Source/Core/DotSpatial.NTSExtension/TopologyText.Designer.cs (hunk 4 above)
[the remark this answers is not in the document]
Comment (jany-tenaj) [reply Source/Core/DotSpatial.NTSExtension/TopologyText.Designer.cs:10]: Ok understood. Leave it in. Who knows what problems VS brings up, when you revert it :/

File: Source/Core/DotSpatial.Data.Forms/DotSpatial.Data.Forms.csproj (hunk 5 above)
[the remark this answers is not in the document]
Comment (jany-tenaj) [reply Source/Core/DotSpatial.Data.Forms/DotSpatial.Data.Forms.csproj:42]: Ok so we'll leave those. I'm currently wondering why everything compiled ok for me... Can't say I understand Vs 2022 ;)

File: Source/.nuget/Pack.bat (hunk 1 above)
Comment (jany-tenaj) [reply Source/.nuget/Pack.bat:22]: @inforithmics Any comment here? This is the last thing keeping me from merging.

Comment (inforithmics) [reply Source/.nuget/Pack.bat:22]: The reason I changed to .nuspec was that with csrpoj and .nuspec combined the nuget package creation didn't work.
When the goal ist to get rid of .nuspec I can try if it works without .nuspec or a reduced one.

Comment (jany-tenaj) [reply Source/.nuget/Pack.bat:22]: Based on the fact that VS2022 is able to build nupkgs from csproj I'd love to get rid of the nuspecs. If you have the time I'd be grateful if you'd take a look at that :)
